### PR TITLE
Init app

### PR DESCRIPTION
**Pull request by SusieSJP**
Hey, I have just set up the sign in and login page, please review.

### CODE REVIEW

File: smart-goals-app/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# SmartGoalsApp
+# SmartGoalsApp2
```
Comment (ChicoLin) [inline smart-goals-app/README.md]: Rename to SmartGoalsApp

File: smart-goals-app/angular.json [hunk 2]
```diff
@@ -3,20 +3,17 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "smart-goals-app": {
+    "smart-goals-app2": {
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: rename (remove "2")

File: smart-goals-app/angular.json [hunk 3]
```diff
@@ -3,20 +3,17 @@
...
+      "schematics": {},
       "root": "",
       "sourceRoot": "src",
       "prefix": "app",
       "architect": {
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist/smart-goals-app",
+            "outputPath": "dist/smart-goals-app2",
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: remove 2

File: smart-goals-app/angular.json [hunk 4]
```diff
@@ -68,18 +63,18 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "smart-goals-app:build"
+            "browserTarget": "smart-goals-app2:build"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: same here

File: smart-goals-app/angular.json [hunk 5]
```diff
@@ -68,18 +63,18 @@
...
           },
           "configurations": {
             "production": {
-              "browserTarget": "smart-goals-app:build:production"
+              "browserTarget": "smart-goals-app2:build:production"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: same here

File: smart-goals-app/angular.json [hunk 6]
```diff
@@ -68,18 +63,18 @@
...
             }
           }
         },
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "smart-goals-app:build"
+            "browserTarget": "smart-goals-app2:build"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: ditto

File: smart-goals-app/angular.json [hunk 7]
```diff
@@ -117,21 +111,15 @@
           "builder": "@angular-devkit/build-angular:protractor",
           "options": {
             "protractorConfig": "e2e/protractor.conf.js",
-            "devServerTarget": "smart-goals-app:serve"
+            "devServerTarget": "smart-goals-app2:serve"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: ditto

File: smart-goals-app/angular.json [hunk 8]
```diff
@@ -117,21 +111,15 @@
...
           },
           "configurations": {
             "production": {
-              "devServerTarget": "smart-goals-app:serve:production"
+              "devServerTarget": "smart-goals-app2:serve:production"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: ditto

File: smart-goals-app/angular.json [hunk 9]
```diff
@@ -117,21 +111,15 @@
...
-  },
-  "defaultProject": "smart-goals-app",
-  "schematics": {
-    "@schematics/angular:component": {
-      "styleext": "scss"
-    }
-  }
-}
+    }},
+  "defaultProject": "smart-goals-app2"
```
Comment (ChicoLin) [inline smart-goals-app/angular.json]: remove "2"

File: smart-goals-app/e2e/src/app.e2e-spec.ts [hunk 10]
```diff
@@ -10,7 +10,7 @@ describe('workspace-project App', () => {
 
   it('should display welcome message', () => {
     page.navigateTo();
-    expect(page.getTitleText()).toEqual('smart-goals-app app is running!');
+    expect(page.getTitleText()).toEqual('smart-goals-app2 app is running!');
```
Comment (ChicoLin) [inline smart-goals-app/e2e/src/app.e2e-spec.ts]: remove "2"

File: smart-goals-app/karma.conf.js [hunk 11]
```diff
@@ -16,7 +16,7 @@ module.exports = function (config) {
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
     coverageIstanbulReporter: {
-      dir: require('path').join(__dirname, './coverage/smart-goals-app'),
+      dir: require('path').join(__dirname, './coverage/smart-goals-app2'),
```
Comment (ChicoLin) [inline smart-goals-app/karma.conf.js]: remove "2"

File: smart-goals-app/package.json [hunk 12]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "smart-goals-app",
+  "name": "smart-goals-app2",
```
Comment (ChicoLin) [inline smart-goals-app/package.json]: remove "2"

File: smart-goals-app/src/app/app.component.spec.ts [hunk 13]
```diff
@@ -20,16 +20,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'smart-goals-app'`, () => {
+  it(`should have as title 'smart-goals-app2'`, () => {
```
Comment (ChicoLin) [inline smart-goals-app/src/app/app.component.spec.ts]: remove "2"

File: smart-goals-app/src/app/app.component.spec.ts [hunk 14]
```diff
@@ -20,16 +20,16 @@ describe('AppComponent', () => {
...
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.debugElement.componentInstance;
-    expect(app.title).toEqual('smart-goals-app');
+    expect(app.title).toEqual('smart-goals-app2');
```
Comment (ChicoLin) [inline smart-goals-app/src/app/app.component.spec.ts]: remove "2"

File: smart-goals-app/src/app/app.component.spec.ts [hunk 15]
```diff
@@ -20,16 +20,16 @@ describe('AppComponent', () => {
...
   });
 
   it('should render title', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.querySelector('.content span').textContent).toContain('smart-goals-app app is running!');
+    expect(compiled.querySelector('.content span').textContent).toContain('smart-goals-app2 app is running!');
```
Comment (ChicoLin) [inline smart-goals-app/src/app/app.component.spec.ts]: remove "2"

File: smart-goals-app/src/app/app.component.ts [hunk 16]
```diff
@@ -1,11 +1,12 @@
...
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css'],
-  encapsulation: ViewEncapsulation.None
+  styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'smart-goals-app';
+  title = 'smart-goals-app2';
```
Comment (ChicoLin) [inline smart-goals-app/src/app/app.component.ts]: remove "2"

File: smart-goals-app/src/index.html [hunk 17]
```diff
@@ -2,15 +2,14 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>SmartGoalsApp</title>
+  <title>SmartGoalsApp2</title>
```
Comment (ChicoLin) [inline smart-goals-app/src/index.html]: remove "2"

File: smart-goals-app/angular.json (hunk 2 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 3 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 4 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 5 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 6 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 7 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/angular.json (hunk 9 above)
Comment (SusieSJP) [reply smart-goals-app/angular.json]: removed

File: smart-goals-app/e2e/src/app.e2e-spec.ts (hunk 10 above)
Comment (SusieSJP) [reply smart-goals-app/e2e/src/app.e2e-spec.ts]: removed

File: smart-goals-app/karma.conf.js (hunk 11 above)
Comment (SusieSJP) [reply smart-goals-app/karma.conf.js]: removed

File: smart-goals-app/package.json (hunk 12 above)
Comment (SusieSJP) [reply smart-goals-app/package.json]: removed

File: smart-goals-app/src/app/app.component.spec.ts (hunk 13 above)
Comment (SusieSJP) [reply smart-goals-app/src/app/app.component.spec.ts]: removed

File: smart-goals-app/src/app/app.component.spec.ts (hunk 14 above)
Comment (SusieSJP) [reply smart-goals-app/src/app/app.component.spec.ts]: removed

File: smart-goals-app/src/app/app.component.spec.ts (hunk 15 above)
Comment (SusieSJP) [reply smart-goals-app/src/app/app.component.spec.ts]: removed

File: smart-goals-app/src/app/app.component.ts (hunk 16 above)
Comment (SusieSJP) [reply smart-goals-app/src/app/app.component.ts]: removed

File: smart-goals-app/src/index.html (hunk 17 above)
Comment (SusieSJP) [reply smart-goals-app/src/index.html]: removed

**Comment by SusieSJP** [comment]
removed

**Pull request merged**